ardere/step_functions.py

```diff
@@ -70,18 +90,34 @@ class PlanValidator(Schema):
     name = fields.String(required=True)
     metrics_options = fields.Nested(MetricsOptions, missing={})
 
+    name_validator = re.compile("([:\*]+)")
```

Comment:
Same issue as above, maybe put this regex up at module scope near the max name len?